front/pages/w/[wId]/u/chat/index.tsx

```diff
@@ -73,23 +78,30 @@ export const getServerSideProps: GetServerSideProps<{
     };
   }
 
-  const dataSources = dsRes.value;
+  const dataSources: DataSource[] = dsRes.value.map((ds) => {
+    return {
+      name: ds.name,
+      description: ds.description,
+      provider: ds.connectorProvider || "none",
+      selected: ds.connectorProvider ? true : false,
+    };
+  });
 
-  const managedDataSources = dataSources
-    .filter((ds) => ds.connectorProvider)
-    .map((ds) => {
-      if (!ds.connectorProvider) {
-        throw new Error("provider not defined for the data source");
-      }
-      return {
-        name: ds.name,
-        provider: ds.connectorProvider,
-        selected: true,
-        logoPath: PROVIDER_LOGO_PATH[ds.connectorProvider],
-      };
-    });
+  // Select Data Sources if none are managed.
+  if (!dataSources.some((ds) => ds.provider !== "none")) {
```

Comment:
The idea here is that if we have managed data sources we just want to look at those?